certified-connectors/ZellisIntelligencePlatform/README.md

```diff
@@ -0,0 +1,44 @@
+## Zellis Intelligence Platform Connector
+An official connector working with the Zellis Intelligence Platform.
```

Comment:
@basha-k This vague description of your connector is not sufficient and please provide a detailed description for your connector which everyone can understand.

Reply:
Implemented and changed the description appropriately.